frappe/public/js/frappe/list/list_sidebar.js

```diff
@@ -83,6 +83,14 @@ frappe.views.ListSidebar = class ListSidebar {
 			this.sidebar.find('.list-link[data-view="Image"]').removeClass('hide');
 			show_list_link = true;
 		}
+		// show map link if map_view
+		if ((this.list_view.meta.fields.find(i => i.fieldname === "latitude") &&
+            this.list_view.meta.fields.find(i => i.fieldname === "longitude")) ||
+            (JSON.stringify(frappe.listview_settings) !== '{}' &&
+            frappe.listview_settings[this.list_view.doctype].get_coords_method)) {
```

Comment:
Just a suggestion on the comment and on the conditions order to make it more clear that latitude / longitude are just the fallback solution.
```suggestion
		// show map link if map_view doctype has get_coords or latitude and longitude
		if ((JSON.stringify(frappe.listview_settings) !== '{}' &&
            frappe.listview_settings[this.list_view.doctype].get_coords_method) || 
            (this.list_view.meta.fields.find(i => i.fieldname === "latitude") &&
            this.list_view.meta.fields.find(i => i.fieldname === "longitude"))) {
```